lib/chef/resource/windows_task.rb

```diff
@@ -116,6 +116,9 @@ class WindowsTask < Chef::Resource
                  introduced: "14.4", default: false,
                  description: "Scheduled task option when system is switching on battery."
 
+        property :description, String,
```

Comment:
We need an introduced field here so we can pick this up in our documentation with the version of chef we ship this in. This will probably end up in Chef 14.7

Reply:
Have added the `introduced` field.
Thanks